### PR TITLE
added details of changes and steps

### PR DESCRIPTION
**Pull request by HussainKhanSherwani**
# Pull Request Template

Before submitting this PR, please make sure you have completed the following checklist. This helps ensure that our codebase remains high quality and that our features work well across all supported devices and themes.

### Code Review
- [ ] I have carefully reviewed my code and believe it is final from my side.

### Functional Testing
- [ ] I have thoroughly tested the functionality of my changes.
- [ ] I am confident that my changes do not negatively impact existing features.

### Testing Documentation
Please provide a brief description of the tests you have performed to verify your changes. Include any relevant scenarios or edge cases.

- Example test 1: Tested feature X by uploading the image
- Example test 2: Tested feature X by doing these things.

# Demo Video
Please provide a demo video of your changes. This helps reviewers understand the changes and verify the functionality.

You can record the video using tools like Loom(https://www.loom.com/), Clip by ClickUp(https://clickup.com/features/clips), Sendspark(https://www.sendspark.com/), etc.

Include the link to the video below and make sure the video covers
- [ ] Explain the task/bug you are solving
- [ ] Show the changes in the codebase
- [ ] Show the changes you tested to make sure it works

### CODE REVIEW

File: docs/src/003_z01_generating_reports_reits.md [hunk 1]
```diff
@@ -1,31 +1,49 @@
...
+- The thrid step will be to send this data to flask api
+- The fourth step will be to save the sent dat in agent_status file at specified path
+
+## changes on insights_ui
+
+- we will be adding a new path public-equities/common/ where we will be having the above page at public-equities/common/create
+
+## changes on flask api
+
+- we will add a new api app.py file at path crowd-fund-analysis/public-equities/US/ which will have routes for this project
```
Comment (RobinNagpal) [inline docs/src/003_z01_generating_reports_reits.md:47]: We might try to use another file

Comment (HussainKhanSherwani) [reply docs/src/003_z01_generating_reports_reits.md:47]: sure
